challenge.rb

```diff
@@ -1,26 +1,25 @@
 
 def capitalize_each_string(input)
-  #implement your solution here
+  input.map{|x| x.capitalize}
```

Comment:
If you want to make this even shorter, you can pass a reference to the `capitalize` method instead of a block:
``` ruby
arr.map(&:capitalize)
```